Makefile

```diff
@@ -0,0 +1,95 @@
+# Copyright 2016 The Rook Authors. All rights reserved.
```

Comment:
@BlaineEXE this comment is still valid ;)

Reply:
I'm not sure I can do this. I started with a makefile that ran integration tests and then added their content to that with changes.

Reply:
I'm really sorry. I don't get the issue you have. Why can't you put the original make in the inital commit to keep the original one and make a commit with the diff against the actual one to see your changes.
Today we have just a modified makefile where we can't compute the changes you made.

Reply:
There's no logical connection between the original rook makefile and the one adapted for ceph-container; I don't see why commit history would be relevant.

Reply:
that would make sense if there were any logical connection between the original rook makefile and the one adapted here for ceph-container, but, there isn't

Reply:
I think I agree with @dmick here, but my commits already are set up as requested by @ErwanAliasr1 . I do think that we should keep the Rook copyright; I used enough of the Rook-contributed code (or architecture) that I think it would plagiarism not to acknowledge them.